jquery.floatThead.js

```diff
@@ -705,6 +706,13 @@
             }
             left = tableOffset.left - windowLeft;
           }
+          if (top < 0) {
+            // in some fixed layout's the top offset can return a negative, in which case using
+            // the last known good value works 100%
+            top = goodTop;
```

Comment:
In my case I am using a container with a fixed offset so setting it to 0 would
place the header in the wrong place:
.content-body position : fixed; top : 110px; padding-right : 5px; overflow : auto; }

I see the PR also updated with my updated bower.json - Let me know if you want
me to remove it.




On Sun, Mar 20, 2016 at 4:50 PM, Misha Koryak < notifications@github.com > wrote:
In jquery.floatThead.js :

> @@ -705,6 +706,13 @@
>              }
 
>            }
> +          if (top < 0) {
> +            // in some fixed layout's the top offset can return a negative, in which case using
> +            // the last known good value works 100%
 


wouldnt it be better to set it to 0 here? and then you can remove most of this
logic

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub